module-services/service-evtmgr/screen-light-control/ScreenLightControlParameters.hpp

```diff
@@ -8,6 +8,18 @@
 
 namespace screen_light_control
 {
+    // Application which send light request
+    enum class Application
```

Comment:
I've changed the name and now there is `Sender`. I can't use names from `Application` because one application is too general. The alarm application can send messages such as `pre-wake up` or `snooze`. I hope this implementation is much clear now.

Reply:
Sure, now it's good - before this looked just like a repeating the names with an enum.